### PR TITLE
Add The month in Montage

### PR DESCRIPTION
**Pull request by Stuk**

### CODE REVIEW

**Comment by aadsm** [comment]
The `Alternation` was not added, it's still a work in progress.

**Comment by Stuk** [comment]
Ah, I see, it wasn't merged into master.

**Comment by Stuk** [comment]
Forced pushed some changes.

File: _posts/2014-01-22-the-month-in-montage.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+In Digit a new [transparent skin](https://github.com/montagejs/digit/pull/59) was added that can be easily customized by chaging the `color` CSS property. This is along with the [Button--glass variation](https://github.com/montagejs/digit/pull/24) that makes borderless icon buttons easier. As in the Repetition, [`objectAtCurrentIteration` was deperecated in BigList](https://github.com/montagejs/digit/pull/58). The [tab/focus support](https://github.com/montagejs/digit/pull/18) was improved and support was added for [using an `<a>` element with Button](https://github.com/montagejs/digit/pull/17). Improvement of [the JSDocs continues](https://github.com/montagejs/digit/pull/49).
+
+Joey has been updated to the [latest Q-IO](https://github.com/montagejs/joey/commit/c8c55a1b79378dd37cf14d97beab0ed423c6401b) that fixes host negotiation decision tracking and ranged HTTP requests for files, and the "binary" now supports [priviledge descalation with the `-u UID`argument](https://github.com/montagejs/joey/commit/4d3d7ece0a38c1e0604ea1be39b68166d7f4d5e4). Minit was updated to take advantage of this new release.
```
Comment (kriskowal) [inline _posts/2014-01-22-the-month-in-montage.md:18]: You don’t miss much, @stuk.

Comment (Stuk) [inline _posts/2014-01-22-the-month-in-montage.md:18]: I am the Montage homepage's all-seeing-eye.

Comment (Stuk) [inline _posts/2014-01-22-the-month-in-montage.md:18]: FYI I actually use [this script](https://github.com/Stuk/montage-report).

Comment (kriskowal) [inline _posts/2014-01-22-the-month-in-montage.md:18]: Bravo!

**Pull request merged**